requirements/static/linux.in

```diff
@@ -16,6 +16,7 @@ jxmlease
 kazoo
 keyring==5.7.1
 kubernetes<4.0
+libnacl==1.4.0
```

Comment:
We need to recompile the static requirements if we change these `.in` files

Reply:
Yup, it's waiting on me.

Reply:
Updated